torch/csrc/jit/python/python_arg_flatten.cpp

```diff
@@ -68,7 +72,7 @@ void flatten_rec(PyObject* obj, ParsedArgs& args) {
     args.vars.push_back(var);
     args.desc.metadata.emplace_back(var);
     args.desc.structure.push_back(D::Variable);
-  } else if (strcmp(THPUtils_typename(obj), "NoneType") == 0) {
+  } else if (PyNone_Check(obj)) {
```

Comment:
This change does not seem to be specific to a PR, is it? If so, can it be moved to a separate one?

Reply:
cc original author @garymm

Reply:
@BowenBao do you want me to open another PR in `onnx_ms_1` that reverts the changes in this file? Or how shall I proceed?

Reply:
@garymm is it indeed not specific to this PR? Personally I do not prefer separation, since this PR already touches a lot of other files outside of ONNX as well. 

If there is a need to separate, I think the process could be you directly work on the branch of this PR, and push new commits to this branch. The separated change will be another PR submitted against master.

Reply:
I think it's an unrelated clean-up, but since we're getting a TorchScript code owner to review the PR anyways, I don't see any value in removing this from the PR.

Marking this thread resolved. If the TorchScript owner asks for this to be separated I can do so.